### PR TITLE
Bl or573 mk

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**
Please not then in some cases I have taken data from the mss. directly and not from the catalogue.

### CODE REVIEW

**Comment by smaugustine** [comment]
ms_i1.52 is missing a ref, it's Chaîne no. 237 (`LIT2956RepCh237`).

I had this MS in my own database of Malkǝʾa Gubāʾe MSS and contents, along (partially or fully) with the others (575, 578, etc.), sorry for not mentioning it.

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 1]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="11v"/>
+<title xml:lang="gez" type="complete" ref="LIT2444Temher">Tǝmhǝrta Ḫǝbuʾāt</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="16r"/>
+<title xml:lang="gez" type="complete" ref="LIT2444Temher">Hymn</title>
+<incipit>ሰላም፡ ለሕማምከ፡ ቤተ፡ አይሁድ፡ ዘአንደደ፡ ወረሰዮሙ፡ ሐመደ።</incipit>
+<textLang mainLang="gez"/>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:62]: I guess it is LIT2741RepCh25

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 2]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="23r"/>
+<title xml:lang="gez" type="complete" ref="LIT2842RepCh125"><foreign xml:lang="gez">መልክአ፡ መስቀል፡</foreign></title>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:67]: just put xml:lang into the title element, and you do not need foreign

File: LondonBritishLibrary/orient/BLorient573.xml (hunk 1 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient573.xml:62]: Please, add xml:lang to the incipit here and below

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 3]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<title xml:lang="gez" type="complete" ref="LIT2733RepCh17">The gathering of the faithful</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.7">
+<locus from="37r"/>
+<title xml:lang="gez" type="complete" ref="LIT3034RepCh314"><foreign xml:lang="gez">መልክአ፡ ልሳን፡</foreign></title>
+<incipit>ናሁ፡ ተማኅፀንኩ፡ በኆኅያተ፡ ስምከ፡ ካፍ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:90]: Same here and below

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 4]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<msItem xml:id="ms_i1.13">
+<locus from="70r"/>
+<title xml:lang="gez" type="complete" ref="LIT2847RepCh129b">To the angel Raphael</title>
+<incipit>ሰላም፡ ለዝክረ፡ ስምከ፡ ውስተ፡ አፈ፡ ኵሉ፡ ጥዑም።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="75r"/>
+<title xml:lang="gez" type="complete" ref="LIT2999RepCh279">To S. John the Evangelist</title>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:135]: St

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 5]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="86r"/>
+<title xml:lang="gez" type="complete" ref="LIT2816RepCh100">To the 12 Apostles</title>
+<incipit>ሰላም፡ ለዝክረ፡ ስምክሙ፡ ለለ፩ዱ። ዓሥሩ፡ ወክልኤ፡ ዘይከውን፡ መጠነ፡ ፍቅዱ። </incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="89r"/>
+<title xml:lang="gez" type="complete" ref="LIT2906RepCh187">To SS. Peter and Paul</title>
+<incipit>ሰላም፡ ለዝክረ፡ ስምክሙ፡ ዘአልባስጥሮስ፡ አፈው።</incipit>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:158]: St

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 6]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="104r"/>
+<title xml:lang="gez" type="complete" ref="LIT2875RepCh156">To the angel Gabriel</title>
+<incipit>ሰላም፡ ለዝክረ፡ ስምከ፡ ዘተጸውዓ፡ ቀዲሙ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.21">
+<locus from="111r"/>
+<title xml:lang="gez" type="complete" ref="LIT2866RepCh147">To S. George</title>
+<incipit>ሰላም፡ ለዝክረ፡ ስምከ፡ ዘሰሌዳ፡ ሞገስ፡ መጽሐፉ።</incipit>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:186]: St

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 7]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="149v"/>
+<title xml:lang="gez" type="complete" ref="LIT1864Malkea">To the Blessed Virgin Mary</title>
+<incipit>ሰላም፡ ለስዕልኪ፡ ዘየዕዔድም፡ ንጻሬ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.28">
+<locus from="151r"/>
+<title xml:lang="gez" type="complete" ref="LIT2972RepCh253">To S. Stephen</title>
+<incipit>በስመ፡ ሥሉስ፡ ቅዱስ፡ እግዚአብሔር፡ አሐዱ።</incipit>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:235]: St

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 8]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="157v"/>
+<title xml:lang="gez" type="complete" ref="LIT3075RepCh355">To Sebastian</title>
+<incipit>እንዘ፡ እፈቅድ፡ ፈቂደ፡ ወእትሜነይ፡ ተምኔተ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.30">
+<locus from="161r"/>
+<title xml:lang="gez" type="complete" ref="LIT2924RepCh205">To S. Bartholomew for the 1st of Maskaram</title>
+<incipit>ሰላም፡ ለፅንሠትከ፡ እንበለ፡ ያእምሩ፡ አዝማዲከ።</incipit>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:249]: St here and below

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 9]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<msItem xml:id="ms_i1.2">
+<locus from="11v"/>
+<title xml:lang="gez" type="complete" ref="LIT2444Temher">Tǝmhǝrta Ḫǝbuʾāt</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="16r"/>
+<title xml:lang="gez" type="complete" ref="LIT2444Temher">Hymn</title>
+<incipit>ሰላም፡ ለሕማምከ፡ ቤተ፡ አይሁድ፡ ዘአንደደ፡ ወረሰዮሙ፡ ሐመደ።</incipit>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:61]: here the xml:lang="gez" in the title is wrong, it is English. The same applies for many cases below

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 10]
```diff
@@ -54,49 +429,63 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                    <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                <q xml:lang="gez">ጉባኤ፡ መልክ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+                                </item>
+
+<item xml:id="a2">
+<locus target="#243r"></locus>
+<desc type="OwnershipNote">The note states that the manuscript belonged to <persName role="owner" ref="PRS13311TaklaIyasus">Takla ʾIyasus</persName>
+and his wife <persName role="owner" ref="PRS13312LahyaDengel">Lāhya Dǝngǝl</persName>. The name of Takla ʾIyasus has been erased throughout, but is still
+legible e. g. on <locus target="#231v">f. 231v</locus>.</desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:469]: I guess the information that the name of the owner has been erased should be added to extras. BTW, you do need to write out the names, if they are marked up they will be retrieved from the corresponding file

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 11]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<incipit xml:lang="gez">ሰላም፡ ለሕማምከ፡ ቤተ፡ አይሁድ፡ ዘአንደደ፡ ወረሰዮሙ፡ ሐመደ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="23r"/>
+<title xml:lang="gez" type="complete" ref="LIT2842RepCh125">መልክአ፡ መስቀል፡</title>
+<incipit>ሰላም፡ ለዝክረ፡ ስምከ፡ በመጽሔተ፡ መስቀል፡ ዘተለክዓ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:70]: xml: lang in incipit

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 12]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="168v"/>
+<title xml:lang="en" type="complete" ref="LIT2855RepCh137">To St Philip for the 18th of Ḫǝdar</title>
+<incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ወለስእርትከ፡ ሠርጐ፡ ርእስ።</incipit>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.34">
+<locus from="170v"/>
+<title xml:lang="engez" type="complete" ref="LIT2929RepCh210">To St Andrew for the 4th of Tāḫśāś</title>
+<incipit xml:lang="gez">ሰላም፡ ለፅንሠትከ፡ ወለልደትከ፡ እማኅፀን።</incipit>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:277]: xml:lang in title? en or gez?

File: LondonBritishLibrary/orient/BLorient573.xml [hunk 13]
```diff
@@ -31,21 +31,396 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="177r"/>
+<title xml:lang="en" type="complete" ref="LIT2918RepCh199">To St Jacob son of Alpheus</title>
+<incipit xml:lang="gez">ሰላም፡ ለፅንሠትከ፡ በሥርዓተ፡ ሰብሳብ፡ ዘተጽሐፈ።</incipit>
+<note><date ref="ethiocal:Yakatit10">For the 10th of Yakātit</date></note>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.37">
+<locus from="179v"/>
+<title xml:lang="en" type="complete" ref="LIT2916RepCh197">To S. Matthias(<foreign xml:lang="gez">ማትያስ፡</foreign>) for the 8th of Magābit</title>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient573.xml:298]: St

File: LondonBritishLibrary/orient/BLorient573.xml (hunk 10 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient573.xml:473]: the type is not needed here, it  is rather a phenomena (the name was erased throughout the mansucript) than a specific text, I guess

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient573.xml (hunk 1 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient573.xml:62]: Sorry, I can not see the changes here, or is my assumption wrong?

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**